core/src/main/resources/defaults/devclustertemplates/7.2.15/aws/dataengineering.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "[DEV] 7.2.15 - Data Engineering for AWS",
+  "description": "",
+  "type": "DATAENGINEERING",
+  "featureState": "RELEASED",
+  "cloudPlatform": "AWS",
+  "distroXTemplate": {
+    "cluster": {
+      "blueprintName": "[DEV] 7.2.15 - Data Engineering: Apache Spark, Apache Hive, Apache Oozie"
+    },
+    "instanceGroups": [
+      {
+        "name": "gateway",
+        "template": {
+          "attachedVolumes": [
+            {
+              "count": 1,
+              "size": 64,
+              "type": "gp2"
+            }
+          ],
+          "aws": {
+            "encryption": {
+              "type": "NONE"
+            }
+          },
+          "instanceType": "m5.2xlarge"
```

Comment:
Is it possible to use smaller instances?

Reply:
of course, to be honest, I haven't tried these with large scale, so I am not sure how low we can go, but we can go a little lower for sure

Reply:
the original is m5.4xlarge, maybe we can with m5.xlarge